### DISCUSSION TITLE
Monolingual base file from other repository

### DISCUSSION THREAD

**Discussion by guillemtp** [WeblateOrg/weblate#5941] [General] [answered]
Is there any option to get the monolingual base language file from other repository?

**My goal**
In our development, one development repository is related to one component in Weblate. We have a centralized repository for translations. I want to avoid to have git submodules in  each repository and only have a main development monolingual file in each one. Each weblate component will be configured with "weblate://" repository and I want to use the base file from the development repository instead of the components' path in the "weblate://" repository. Is it possible?

**Comment by nijel** [comment, marked as answer]
Currently, it is not possible to have some files from another repository.
